packages/guides-restructured-text/src/RestructuredText/Directives/CodeBlockDirective.php

```diff
@@ -101,4 +102,15 @@ private function setCaptionBasedOnOptions(array $options, CodeNode $node): void
 
         $node->setCaption($caption);
     }
+
+    /** @param DirectiveOption[] $options */
+    private function setEmphasizeLinesBasedOnOptions(array $options, CodeNode $node): void
+    {
+        $emphasizeLines = null;
+        if (isset($options['emphasize-lines'])) {
+            $emphasizeLines = (string) $options['emphasize-lines']->getValue();
```

Comment:
maybe we should validate the input?

Reply:
how would you validate it? regular expression?

Reply:
yes, that would do the trick, then log a warning when it is invalid.

Reply:
added a regex